packages/app-cli/app/command-ls.ts

```diff
@@ -29,6 +29,7 @@ class Command extends BaseCommand {
 			['-r, --reverse', _('Reverses the sorting order.')],
 			['-t, --type <type>', _('Displays only the items of the specific type(s). Can be `n` for notes, `t` for to-dos, or `nt` for notes and to-dos (eg. `-tt` would display only the to-dos, while `-tnt` would display notes and to-dos.')],
 			['-f, --format <format>', _('Either "text" or "json"')],
+			['-i, --id', _('Show full ID of items."')],
```

Comment:
```suggestion
			['--always-show-full-id', _('Always show full IDs of items.')],
```

In some cases, the full ID of an item can still be shown, even if this option isn't provided. As such, a more descriptive option name could be helpful.